docs/release-notes/artifacts/pr0301.yaml

```
@@ -0,0 +1,14 @@
+# --- Release notes artifact ----
+
+schema_version: 1
+changes:
+  - title: Parametrize app_name for subordinates in Terraform product
```

Erin clarified the expected style for the title: https://canonical-platform-engineering.readthedocs-hosted.com/latest/engineering-practices/documentation/charm-release-notes/release-notes-templates/#style-considerations

```suggestion
  - title: Parametrized app_name for subordinates in Terraform product
```